index.html

```diff
@@ -1851,39 +1851,279 @@ <h5><code>invokeaction</code></h5>
           </a>).
         </section>
       </section>
+
       <section id="events">
         <h4>Events</h4>
-        <section class="ednote">
+        <p>
+          The Core Profile uses Server-Sent Events [[EVENTSOURCE]] as a
+          mechanism for Consumers to subscribe to events emitted by a Web Thing.
+          Any <code>Form</code> within an <code>Event</code> affordance whose 
+          <code>href</code> member contains a URI with a URI scheme of 
+          <code>http</code> or <code>https</code> is assumed to have a 
+          <code>subprotocol</code> of <code>sse</code>, unless specified
```

Comment:
so, the profile is essentially providing a default value for subprotocol...

Reply:
Yes that was my thinking. But I'm happy to remove that if it's contentious.

Reply:
To follow up, I have now removed this default.